apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Image } from 'react-native';
+import { Text, TextBlack, View } from '@walless/ui';
+
+interface Props {
+	name: string;
+	description: string;
+	love?: number;
+	activeUsers?: number;
+}
+
+const ChooseLayoutCard = (props: Props) => {
+	return (
+		<View className="bg-[#f4f4f4] w-full rounded-b">
+			<View className="h-16 bg-light-gray rounded-t">
+				<Image source={require('./card-image.png')} />
```

Comment:
it is more convenient to put image files under `assets/img` folder, and declare asset resource under [utils/config.js](https://github.com/cocrafts/walless/blob/main/apps/extension/src/utils/config.ts).

and then, use following snippet:
```typescript
import { resources } from 'utils/config';

<Image source={resources.path.to.resource}/>
```